api/Sanitizer.json

```
@@ -0,0 +1,457 @@
+{
+  "api": {
+    "Sanitizer": {
+      "__compat": {
+        "spec_url": "https://wicg.github.io/sanitizer-api/#sanitizer",
+        "support": {
+          "chrome": {
+            "version_added": false,
+            "notes": "Chrome 105 to Chrome 118 (inclusive) supported this interface name with a significantly different specification."
+          },
```

So, what will happen in the future here? Is it this?

```suggestion
          "chrome": [
            {
              "version_added": "200"
            },
            {
              "version_added": "105",
              "version_added": "119",
              "notes": "Chrome 105 to Chrome 118 (inclusive) supported this interface name with a significantly different specification."
            }
          ],
```

---

@Elchi3 I was expecting:

```suggestion
          "chrome": {
            "version_added": "200",
            "notes": "Chrome 105 to Chrome 118 (inclusive) supported this interface name with a significantly different specification."
          },
```

... and one day perhaps the note disappears.

---

Really just requires a decision. I'm happy with this.